diff -urNp -x '*.orig' libedit-20191231-3.1.org/doc/Makefile.am libedit-20191231-3.1/doc/Makefile.am
--- libedit-20191231-3.1.org/doc/Makefile.am	2018-05-25 19:50:45.000000000 +0200
+++ libedit-20191231-3.1/doc/Makefile.am	2022-03-01 23:21:05.335645824 +0100
@@ -52,7 +52,7 @@ EL_MAN_LINKS += \
 install-data-hook: $(EL_MAN_LINKS)
 
 $(EL_MAN_LINKS):
-	(cd $(DESTDIR)$(man3dir) && rm -f $@ && $(LN_S) editline.3 $@)
+	(cd $(DESTDIR)$(man3dir) && rm -f $@ && echo '.so editline.3' > $@)
 
 $(EL_MANS):
 	@if test "$(MANTYPE)" = "mdoc"; then\
