--- libedit-20110802-3.0/configure.ac~	2011-08-02 10:09:05.000000000 +0200
+++ libedit-20110802-3.0/configure.ac	2011-08-21 10:54:16.702923119 +0200
@@ -44,7 +44,8 @@
 EL_MANTYPE
 
 
-AC_CHECK_LIB(terminfo, tgetent,,
+AC_CHECK_LIB(tinfo, tgetent,,
+  [AC_CHECK_LIB(terminfo, tgetent,,
     [AC_CHECK_LIB(termcap, tgetent,,
        [AC_CHECK_LIB(termlib, tgetent,,
            [AC_CHECK_LIB(curses, tgetent,,
@@ -54,6 +55,7 @@
            )]
        )]
     )]
+  )]
 )
 
 
